packages/icon-font/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v2.0.0 (Unreleased)
+
+### Changed
+
+-   Changed import path of icon font files to `icons`.
+
+```sh
+import '@brightlayer-ui/icons/BrightlayerUIIcons.css'
```

Comment:
Show a before/after code snippet here using the 'diff' language and show it for both react and angular.

e.g.,

```diff
- import '@brightlayer-ui/icons/iconfont/BrightlayerUIIcons.css'
+ import '@brightlayer-ui/icons/BrightlayerUIIcons.css'
```